src/sentry/static/sentry/app/views/alerts/builder/builderBreadCrumbs.tsx

```diff
@@ -1,34 +1,43 @@
 import React from 'react';
 import styled from '@emotion/styled';
 
-import Breadcrumbs from 'app/components/breadcrumbs';
+import Breadcrumbs, {Crumb} from 'app/components/breadcrumbs';
 import {t} from 'app/locale';
 import space from 'app/styles/space';
 
 type Props = {
   hasMetricAlerts: boolean;
   orgSlug: string;
   title: string;
+  projectSlug: string;
+  alertName?: string;
 };
 
 function BuilderBreadCrumbs(props: Props) {
-  const {hasMetricAlerts, orgSlug, title} = props;
-  return (
-    <StyledBreadcrumbs
-      crumbs={[
-        {
-          to: hasMetricAlerts
-            ? `/organizations/${orgSlug}/alerts/`
-            : `/organizations/${orgSlug}/alerts/rules/`,
-          label: t('Alerts'),
-          preserveGlobalSelection: true,
-        },
-        {
-          label: title,
-        },
-      ]}
-    />
-  );
+  const {hasMetricAlerts, orgSlug, title, alertName, projectSlug} = props;
+  const crumbs: Crumb[] = [
+    {
+      to: hasMetricAlerts
+        ? `/organizations/${orgSlug}/alerts/`
+        : `/organizations/${orgSlug}/alerts/rules/`,
+      label: t('Alerts'),
+      preserveGlobalSelection: true,
+    },
+    {
+      label: title,
+      ...(alertName
```

Comment:
using an alertName (e.g. `Throughput`, `Issues`, `Number of Errors`) as a proxy for including the wizard, since this will only be given as a prop if the user is coming from the wizard